css/store.css

```diff
@@ -0,0 +1,285 @@
+* {
```

Comment:
Hi @ShadiSafaee, I can see you created 2 pull requests and both of them are for week2 homework and the other PR has been reviewed, is this PR extra?
please let me know. so I can close or review this PR :) 
Thanks

Reply:
Hi @ellietms,

This is extra, created by mistake 
https://github.com/ShadiSafaee/HTML-CSS-Coursework-Week2/pulls

Reply:
Hi @ShadiSafaee, This current PR is extra? (Is this your week2 home work?) 
https://github.com/CodeYourFuture/HTML-CSS-Coursework-Week2/pull/336

Reply:
@ellietms I think this has happened as I created a new branch for this week homework, whatever comes under "store" name is related to this week. So, I`m not sure which one can be deleted as my TA has commented the earlier version and it`s good for me to keep them. Any idea?